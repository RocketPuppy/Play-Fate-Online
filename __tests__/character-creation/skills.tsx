import * as React from "react";
import * as renderer from "react-test-renderer";
import Skills from "../../src/character-creation/skills";

test("renders all the pieces", () => {
  const render = renderer.create(<Skills />);

  expect(render.toJSON()).toMatchSnapshot();
});
