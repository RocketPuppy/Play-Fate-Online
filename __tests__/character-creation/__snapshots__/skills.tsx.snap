// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders all the pieces 1`] = `
<div>
  <h2>
    Skills
  </h2>
  <ul>
    <li>
      One at Great (+4)
    </li>
    <li>
      Two at Good (+3)
    </li>
    <li>
      Three at Fair (+2)
    </li>
    <li>
      Four at Average (+1)
    </li>
  </ul>
</div>
`;
